packages/shared/src/models/IPodman.ts

```diff
@@ -0,0 +1,26 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+export enum VMType {
+  WSL = 'wsl',
+  LIBKRUN = 'libkrun',
+  QEMU = 'qemu',
+  APPLEHV = 'applehv',
+  HYPERV = 'hyperv ',
```

Comment:
Is the extra space expected ?

Reply:
No, I will open a follow up PR to fix this, nice catch !

Reply:
See https://github.com/containers/podman-desktop-extension-ai-lab/pull/1346